src/pages/VaultsPage/HasVaults.tsx

```diff
@@ -241,6 +245,32 @@ export default function HasVaults({ epochs, vault }: HasVaultsProps) {
         deposit={() => setModal('deposit')}
         transactions={vault.transactions}
       />
+      <div>
+        <h3>gqty debugging</h3>
+        org: {org.name}
+        <ul>
+          {org.circles().map(c => {
+            return (
+              <li key={c.id}>
+                {c.name}
+                <ul>
+                  {c
+                    .epochs({
+                      // limit: 2,
+                      // where: { ended: { _eq: false } },
+                      order_by: [{ start_date: order_by.asc }],
+                    })
+                    .map(e => {
+                      <li key={e.id}>
```

Comment:
since on first pass this will be null, you can do this:
```suggestion
                      <li key={e.id || 0}>
```